test/base/binding.coffee

```diff
@@ -63,9 +63,11 @@ describe("Quickdraw.Internal.Binding", ->
             )
 
             errorSpy = sandbox.qd._.errors.throw = sinon.spy()
+            nodePathSpy = sandbox.qd._.binding._getNodePath = sinon.spy()
```

Comment:
Not a big deal here, but just FYI: doing this would replace the underlying `_getNodePath` function with a _new_ anonymous function.

```js
let spy;
let myObj = new MyObj();

// Replaces "foo()" with anonymous spy
spy = myObj.foo = sinon.spy();

// Wraps "foo()"; preserves original functionality
spy = sinon.spy(myObj, foo);
```

Reply:
Yeah, that was intentional, the purpose of this particular check was to see if we invoke _getNodePath, independent of what the actual contents of that function are